src/components/Store.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react'
+import Cart from './Cart'
+import { productsData } from '../data/productsData'
+import ProductsList from './ProductsList'
+
+const Store = () => {
+  const [display, setDisplay] = useState('products')
+  const [products, setProducts] = useState(productsData)
+  const [cartItems, setCartItems] = useState([])
+  
+
+  return (
+    <>
+      <button onClick={ () => setDisplay('products') } > Produtos </button>
+      <button onClick={ () => setDisplay('cart') } > Carrinho </button>
+      { 
+        display === 'products' ? 
+        <ProductsList productsToShow={ products }/> : 
+        <Cart cartItemsToShow={ cartItems } />
+      }
```

Comment:
Para ficar mais explicito quais são as opções, eu faria algo assim:
```suggestion
      {display === 'products' &&  <ProductsList productsToShow={ products }/>}
      {display === 'cart' &&  <Cart cartItemsToShow={ cartItems } />}
```

Mas não está errado do jeito que fez.